src/request.ts

```diff
@@ -284,6 +285,16 @@ class DatastoreRequest {
           readConsistency: code,
         };
       }
+      if (options.readTime) {
+        if (!reqOpts.readOptions) {
```

Comment:
nit:

I might make this explicitly, `if (reqOpts.readOptions !== undefined)`.

Since JavaScript is so loose with its truthiness, I like to be explicit.

Reply:
Changed to `if (reqOpts.readOptions === undefined) {`